website/docs/reference/node-selection/methods.md

```diff
@@ -173,7 +173,7 @@ dbt test --select "test_type:singular"       # run all singular tests
 
 The `test_name` method is used to select tests based on the name of the generic test
```

Comment:
We will want to come back to this page as well once we add the docs for unit testing!